internal/builders/istio.go

```diff
@@ -10,8 +10,9 @@ import (
 )
 
 const (
-	istioIngressGatewayPrincipal      string = "cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"
-	oathkeeperMaesterAccountPrincipal string = "cluster.local/ns/kyma-system/sa/oathkeeper-maester-account"
+	istioIngressGatewayPrincipal        string = "cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"
+	oathkeeperMaesterAccountPrincipal   string = "cluster.local/ns/kyma-system/sa/oathkeeper-maester-account"
+	AuthorizationPolicyAppSelectorLabel string = "app"
```

Comment:
```suggestion
	authorizationPolicyAppSelectorLabel string = "app"
```
I think we don't need to export this.